src/Gui/DlgSettingsDocument.ui

```diff
@@ -476,7 +476,7 @@ Common sizes are 128, 256 and 512</string>
         <item>
          <widget class="Gui::PrefSpinBox" name="prefCountBackupFiles">
           <property name="minimum">
-           <number>1</number>
+           <number>5</number>
           </property>
```

Comment:
```suggestion
           <number>1</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
```
I didn't tested so you still have some homework to do. :) 
Also you have to update the default value at https://github.com/FreeCAD/FreeCAD/blob/4d395f85bbb9ae595978669fe185332a29cf755e/src/App/Document.cpp#L2681

Reply:
Thanks! Done in 4646202